README.md

```diff
@@ -339,6 +361,13 @@ using (IDbConnection conn = new SnowflakeDbConnection())
 }
 ```
 
+Close the Connection
+--------------------
+
+To close the connection, call the `Close` method of `SnowflakeDbConnection`.
+
+If you want to avoid blocking threads while the connection is closing, call the `CloseAsync` method instead.
```

Comment:
Do we need to explain the CancellationToken argument?
Is it this?
https://docs.microsoft.com/en-us/dotnet/api/system.threading.cancellationtoken?view=net-5.0

Reply:
CancellationToken is a well know async parameter. It give the user a token which can be used to cancel an asynchronous method if needed. I think it's Ok and it doesn't need an extra explanation.

Now, since this is another method which is not readily available in IDbConnection, the user need to make sure that he is using a SnowflakeDbConnection.

Reply:
Thanks! I added a note to tell readers to cast before calling the method